model/import/CsvImportForm.php

```diff
@@ -26,13 +26,18 @@
 use tao_helpers_form_FormFactory;
 use tao_helpers_form_xhtml_Form;
 use tao_models_classes_import_CsvUploadForm;
+use oat\oatbox\service\ServiceManager;
+use Zend\ServiceManager\ServiceLocatorInterface;
+use oat\tao\helpers\UrlHelper;
 
 class CsvImportForm extends tao_models_classes_import_CsvUploadForm
 {
-    public function __construct(array $data = [], array $options = [])
-    {
-        $options[tao_models_classes_import_CsvUploadForm::IS_OPTION_FIRST_COLUMN_ENABLE] = false;
+    private $uri;
```

Comment:
Missing PHP doc to tell this is a string. We could also rename it to classUri do be more accurate